datasets/flwr_datasets/partitioner/__init__.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2023 Flower Labs GmbH. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Flower Datasets Partitioner package."""
+
+
+from .iid_partitioner import IidPartitioner
+from .partitioner import Partitioner
+
+__all__ = [
+    "IidPartitioner",
+    "Partitioner",
+]
```

Comment:
This is new. I got some mypy error without it while importing directly from paritioner. e.g. paritioner.IidParitioner.

Reply:
In other places, we had to use `from .partitioner import Partitioner as Partitioner`

Reply:
What is the reason for that? I no longer get the mypy errors e.g. (in `fds-v0.0.1` branch)
```flwr_datasets/utils.py:20: error: Module "flwr_datasets.partitioner" does not explicitly export attribute "IidPartitioner"  [attr-defined]```
and I kept the imports like this
```from flwr_datasets.partitioner import IidPartitioner, Partitioner```

Reply:
If I have
```from .partitioner import IidPartitioner, Partitioner```
no problems either